ui/app/components/clients/horizontal-bar-chart.js

```diff
@@ -0,0 +1,200 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { assert } from '@ember/debug';
+import { stack } from 'd3-shape';
+// eslint-disable-next-line no-unused-vars
```

Comment:
what's the unused var?

Reply:
for some reason eslint underlines `selectAll` even though it _is_ used - so the comment just tells linter to ignore 
<img width="438" alt="Screen Shot 2021-12-07 at 4 31 43 PM" src="https://user-images.githubusercontent.com/68122737/145127020-f09f0f9c-bb8c-463e-8c37-a70f461e79c5.png">